test/Microsoft.AspNetCore.Authentication2.Test/Google/GoogleMiddlewareTests.cs

```diff
@@ -1100,7 +1103,7 @@ private static TestServer CreateServer(Action<GoogleOptions> configureOptions, F
                         };
                     });
                     services.AddCookieAuthentication(TestExtensions.CookieAuthenticationScheme);
-                    services.AddGoogleAuthentication(configureOptions);
+                    //services.AddGoogleAuthentication(configureOptions);
```

Comment:
Mixing old and new patterns of registration also don't matter since order no longer has any real meaning to the schemes...